vmnet/vmnet.go

```diff
@@ -5,9 +5,10 @@ package vmnet
 import (
 	"fmt"
 	"net"
-	"os"
 	"os/exec"
 	"strings"
+
+	"github.com/docker/machine/libmachine/mcnutils"
```

Comment:
Why add `"github.com/docker/machine/libmachine/mcnutils"`?
https://github.com/docker/machine/blob/master/libmachine/mcnutils/utils.go is not have IsExist()